docs/quickstart.rst

```diff
@@ -271,5 +271,50 @@ the Managed Services Tooling bootstrap image.
    If you used the parameters from the example above, the created ISO image
    will be placed in ``~/build/ms-centos-7.iso``
 
-#. Deploy an OSS Tooling appliance using the created ISO image.
+.. _qs-automated-deployment:
 
+Automated deployment of the OSS infrastructure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The deployment of OSS Tools requires at least three hardware or virtual nodes
+which are connected in the same network. These nodes have to be installed from
+the pre-built OSS Tooling ISO.
+
+.. note::
+    During the installation from ISO the `mirantis` user will be created with
+    the password `mirantis`.
+
+After installation of nodes `automation` have to be copied on one of the nodes:
+
+    ::
+
+      git clone https://github.com/seecloud/automation
+
+Then, change the current directory to `automation` and create an inventory
+configuration for the installed nodes like this:
+
+    ::
+
+      utils/inventory-generator --nodes \
+        node1[ansible_ssh_host=10.20.0.1]
+        node2[ansible_ssh_host=10.20.0.2]
+        node3[ansible_ssh_host=10.20.0.3]
+
+The command above generates an inventory file which is located at
+`inventory/inventory.cfg`. This file is used by automation playbooks.
+
+The Internate connection is needed to perform the bootstrap process which
+downloads all necessary images for further installation. Perform the bootstrap:
+
+    ::
+
+      ansible-playbook -i inventory/inventory.cfg bootstrap-runner.yml
```

Comment:
Nope, inventory.cfg is used for actual deployment. Bootstrap is running only on the deployment machine and doesn't require any additional configuration. This is done on idea, bootstrap and deployment can be triggered on different machines with manual moving of the deployment code